### PR TITLE
feat(config): Add support for LINTER_PATH param

### LINKED ISSUE #161: To override LINTER_PATH
**Is your feature request related to a problem? Please describe.**
Most of the existing repo will already have existing lint rules, and most of them will be placed in root directory of project.

Moving all of them to LINTER_PATH (e.g. .github/linters) wil require either some duplication or breaking exiting way to run lint locally

**Describe the solution you'd like**
To have ability to override this param LINTER_PATH.

**Describe alternatives you've considered**
NA

**Additional context**
NA

### PR DESCRIPTION
**Pull request by sayboras**
To add support for LINT_PATH param. This will make super-lint more
usable for existing projects, which might have lint rule file in
other location. The default value is still .github/lints.

Closes #161

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,6 +113,7 @@ and won't run anything unexpected.
 | **ENV VAR** | **Default Value** | **Notes** |
 | --- | --- | --- |
 | **VALIDATE_ALL_CODEBASE** | `true` | Will parse the entire repository and find all files to validate across all types. **NOTE:** When set to `false`, only **new** or **edited** files will be parsed for validation. |
+| **LINTER_PATH** | `.github/linters` | Directory for all linter configuration rules. |
```
Comment (marbar3778) [inline README.md:116]: IMO, it should look at the root of the project by default. 

Not sure the design choice for requiring `.github/linters` in the first place

Comment (drewish) [reply README.md:116]: I agree that it's a bad default, but @sayboras' choice to preserve it and avoid a breaking change is probably the fastest path to a merge.

**Comment by drewish** [comment]
Needs a rebase due to conflicts in readme

**Comment by GaboFDC** [comment]
I think this will also close https://github.com/github/super-linter/issues/174, https://github.com/github/super-linter/issues/124, and https://github.com/github/super-linter/issues/148

But for the last one, I think file name needs to be changed, we are using rubocop which expects default file `.rubocop.yml` but the linter file is called `.ruby-lint.yml`, WDYT?

**Comment by sayboras** [comment]
Thanks for your review, I have rebased with master

File: README.md (hunk 1 above)
Comment (sayboras) [reply README.md:121]: yup, it's my intention to have a same capability without breaking existing users

**Comment by admiralAwkbar** [comment]
Moving this into a formal branch for full CI

**Pull request closed without merging**